google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeployAction.java

```diff
@@ -221,18 +217,19 @@ public void run() {
     }
 
     private void stop(@NotNull UndeploymentTaskCallback callback) {
-      String moduleToStop;
+      DeployOutput deployOutput;
       try {
-        moduleToStop = parseDeployOutputToModule(deploymentOutput);
```

Comment:
Not that performance is really a concern here but just to explain the original reason I did this: having it in the stop() method lazily does the json parsing (doesn't happen until the user actually tries to undeploy which may never happen in the first place).